utils/middlewares/productName.ts

```diff
@@ -0,0 +1,16 @@
+import { IRequestMiddleware } from "italia-ts-commons/lib/request_middleware";
```

Comment:
I suggest to remove this middleware and just use https://github.com/pagopa/io-functions-commons/blob/master/src/utils/middlewares/required_body_payload.ts#L13 

`RequiredBodyMiddleware(ProductNamePayload)` in the handler